molecule/default/group_vars/all.yml

```diff
@@ -1,8 +1,11 @@
 ---
+consul_version: 1.2.3
```

Comment:
IMO, it should be interesting to test a more modern version

Reply:
Retro compability is broken since this version
https://github.com/hashicorp/consul/blob/master/CHANGELOG.md#140-november-14-2018

Requires manual migration for existing ACLs and python-consul dependency has no support yet either, so needs a deeper conversation to support newer consul versions